### DISCUSSION TITLE
Query error worker call timeout status 400 in budibase mongo query

### DISCUSSION THREAD

**Discussion by Ashraful-Dowla** [Budibase/budibase#4281] [Help]
![WhatsApp Image 2022-01-31 at 8 22 58 PM](https://user-images.githubusercontent.com/44186340/151914665-ebfe0525-3523-4679-823c-22281d11d260.jpeg)
![WhatsApp Image 2022-01-31 at 8 24 05 PM](https://user-images.githubusercontent.com/44186340/151914666-5070143f-0d4e-4f30-8e3b-c2864b5ccd4a.jpeg)
I am having problem with the issue for a long time. I am running both budibase and mongo db in docker. I cannot run my mongo query because of an exception throwing "Query error message:worker call timeout status:400"

**Comment by pavleks** [comment]
Hi @Ashraful-Dowla you could try to change the query timeout limit by adding `QUERY_THREAD_TIMEOUT: 30000,` in your docker-compose.yaml and restart the stack, it might help. I believe it's 10 seconds by default.
